### PR TITLE
grammatical errors in code of conduct

### PR DESCRIPTION
**Pull request by HimanshuMahto**
There are grammatical mistakes in the code of conduct

### CODE REVIEW

**Comment by HimanshuMahto** [comment]
@smanist @wang-chen @jingnanshi Please do review this PR

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ diverse, inclusive, and healthy community.
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our
-community include:
+community includes:
```
Comment (wang-chen) [inline CODE_OF_CONDUCT.md:18]: The subject is `Examples`, not `community`.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ permanent ban.
 sustained inappropriate behavior.
 
 **Consequence**: A temporary ban from any sort of interaction or public
-communication with the community for a specified period of time. No public or
+communication with the community for a specified period. No public or
```
Comment (wang-chen) [inline CODE_OF_CONDUCT.md:101]: `a period of time` is a correct phrase.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## Our Pledge
 
-We as members, contributors, and leaders pledge to make participation in our
+We as members, contributors, and leaders pledge to participate in our
```
Comment (wang-chen) [inline CODE_OF_CONDUCT.md:5]: We use "make participation" to imply "facilitating or ensuring someone's involvement.
This is different from directly "participate" as some roles.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ Examples of unacceptable behavior include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or email
   address, without their explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
+* Other conduct that could reasonably be considered inappropriate in a
```
Comment (wang-chen) [inline CODE_OF_CONDUCT.md:36]: I prefer "which".

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -82,12 +82,12 @@ behavior was inappropriate. A public apology may be requested.
 
 ### 2. Warning
 
-**Community Impact**: A violation through a single incident or series
+**Community Impact**: This violation occurs through a single incident or series
```
Comment (wang-chen) [inline CODE_OF_CONDUCT.md:85]: We don't use "this" to refer to some specific action.

**Review by wang-chen — CHANGES_REQUESTED** [review]

**Comment by HimanshuMahto** [comment]
so should i change them or will you close the PR

**Comment by wang-chen** [comment]
> so should i change them or will you close the PR

Thanks for your PR. Since most changes are not accepted, I will temporarily close this PR.
You may reopen it once you find other necessary changes. Thanks so much for your time!

**Pull request closed without merging**